CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Current Trunk
 - Change constant values of some reference types in the C and JS APIs. This is
   only observable if you hardcode specific values instead of calling the
   relevant methods (like `BinaryenTypeDataref()`). (#4755)
+- `i31ref` and `dataref` are now nullable. (#4843)
```

Comment:
Maybe add that this follows a spec change? (otherwise it seems like maybe it's some decision we made)